Summers_James/detail.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponseRedirect
+from django_mako_plus import view_function, jscontext
+from django import forms
+import re
+from catalog import models as cmod
+import math
+from django.conf import settings
+from datetime import datetime, timezone
+from django.shortcuts import render
+
+
+@view_function
+def process_request(request, prod_id=None):
+    product = cmod.Product.objects.filter(id=prod_id)[0]
+    cName = product.category
+
+    underline = int(cmod.Category.objects.filter(name=cName)[0].id)
```

Comment:
I take it this is named underline because it's the category that will be emphasized in the menu, but if the style of how the category is highlighted were to change to be bold rather than underline it'd be hard to tell what this is used for. If the name were `category_to_underline` or `menu_underline` or something it might make more sense when you come back to the code later.

Reply:
Yes yes yes! I like the clean approach to code 👍